py/desiutil/test/test_io.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""Test desiutil.io.
+"""
+from __future__ import (absolute_import, division,
+                        print_function, unicode_literals)
+# The line above will help with 2to3 support.
+import unittest
+import sys
+import numpy as np
+from warnings import catch_warnings, simplefilter
+import pdb
+from ..io import yamlify
+
+
+class TestIO(unittest.TestCase):
+    """Test desiutil.io
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    @classmethod
+    def tearDownClass(cls):
+        pass
+
+    def test_yamlify(self):
+        """Test yamlify
+        """
+        fdict = {'name':'test', 'num':np.int32(3), 'flt32':np.float32(3.), 'flt64':np.float64(2.),
+                     'num2':np.int64(4), 'bool':np.bool(True), 'lst':['tst2', np.int16(2)],
+                     'tup':(1,3), 'dct':{'a':'tst3', 'b':np.float32(6.)}}
+        assert isinstance(fdict,dict)
```

Comment:
You should use the `assert*` methods of `unittest.TestCase` objects to be fully compatible with `unittest`.  This line should be `self.assertIsInstance(fdict, dict).  Although I'm not exactly sure why you need to test that since it is manifestly a dictionary.